### DISCUSSION TITLE
When does the package get released?

### DISCUSSION THREAD

**Discussion by KowalskiThomas** [benbenbang/types-confluent-kafka#228] [Q&A] [answered]
👋  Hello!

There are some changes in the last commit added upstream that I would like to use in my code. I thought the release process would be automatic, but it doesn't seem to be. 

Is there a schedule or something? Or anything I can help with?

Thank you!

**Comment by benbenbang** [comment, marked as answer]
Hello @KowalskiThomas 

please check [PyPI - 1.3.1](https://pypi.org/project/types-confluent-kafka/1.3.1/)

**Reply by KowalskiThomas** [reply]
Thank you! 🙇
